pallets/unique/Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = 'pallet-unique'
+description = 'Manage Unique Entries.'
+version = '0.8.0'
+authors = ['Dhiway Networks <info@dhiway.com>']
+edition = '2021'
+license = "GPL-3.0"
+homepage = "https://cord.network"
+repository = "https://github.com/dhiway/cord"
+
+[[package.metadata.docs.rs]]
+targets = ['x86_64-unknown-linux-gnu']
+
+[build-dependencies]
+substrate-wasm-builder = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.42" }
+
+[dev-dependencies]
+sp-core = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.42", features = ["std"]}
+sp-keystore = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.42", features = ["std"]}
+pallet-balances = { git = "https://github.com/dhiway/substrate", branch = "release-v0.9.42", features = ["std"]}
+cord-utilities = { package = "cord-utilities", path="../../utilities", features = ["mock"] }
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false, features = [
+	"derive",
+	"max-encoded-len",
+] }
+scale-info = { version = "2.5.0", default-features = false, features = ["derive"] }
+cord-primitives = { package = "cord-primitives", path="../../primitives", default-features = false }
+cord-utilities = { package = "cord-utilities", path = "../../utilities", default-features = false }
+pallet-registry = {  default-features = false, path = '../registry' }
+pallet-schema = {  default-features = false, path = '../schema' }
```

Comment:
This dependency is not required I guess. Similarly review every presence of 'Schema' in the PR.